src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java

```diff
@@ -65,7 +66,7 @@ public AwayCommand message(@Nonnull String message) {
      * @return this command
      */
     @Nonnull
-    public AwayCommand messageRemove() {
+    public AwayCommand notAway() {
```

Comment:
`unaway` would match RFC 1459's name for the numeric - RPL_UNAWAY for that state. The description however (5.1), describes it as "to remove the AWAY message" so perhaps `messageRemove` *was* the best answer?